drivers/storage/portworx/preflight.go

```diff
@@ -410,7 +410,19 @@ func (u *preFlightPortworx) processPassedChecks(recorder record.EventRecorder) {
 		if pxutil.IsVsphere(u.cluster) {
 			cmetaData = DefCmetaVsphere
 		} else if pxutil.IsPure(u.cluster) {
-			cmetaData = DefCmetaFACD
+			var podName string
+		specsItr:
+			for _, spec := range *u.cluster.Spec.CloudStorage.DeviceSpecs {
+				options := strings.Split(spec, ",")
+				for _, option := range options {
+					keyValPair := strings.Split(option, "=")
+					if len(keyValPair) == 2 && keyValPair[0] == "pod" {
+						podName = option
```

Comment:
Let's not name the variable name as podName here. It appears as if it's the name of the pod while it actually is the key, value pair. Maybe let's just suffix it with "Option".

Reply:
We actually have a drive format validation check in px-runc pre-flight.   So we need to make sure that the check in px-runc will accept this format `"size=49,pod=<name>"`

sudo /opt/pwx/bin/px-runc pre-flight -c .... -s size=200,pod=test ... -T px-storeV2 ... --cloud_provider pure ...

Reply:
People did some changes in `px-runc.go` that will be cherry-picked into the px release branch, and I believe this commit serves the purpose of skipping the validation: https://github.com/pure-px/porx/commit/e1d44d49737daf6f11c083a5c228a74c58913579

Reply:
I also cherry-picked this commit to my local test and px didn't complain with the format